fulp_modules/features/antagonists/bloodsuckers/code/powers/hecata/spirit_call.dm

```diff
@@ -0,0 +1,60 @@
+///summon wraiths (weakened shades) to attack anyone who isn't a zombie. This includes non-zombified vassals. However, you can get around this by zombifying your vassals.
+///to do this, you can make someone your favorite vassal, or you can kill them and then revive them with necromancy.
+/datum/action/cooldown/bloodsucker/hecata/spiritcall
+	name = "Spirit Call"
+	desc = "Summon angry wraiths which will attack anyone whose flesh is still alive. Summon amount increases as this ability levels up."
+	button_icon_state = "power_spiritcall"
+	power_explanation = "Spirit Call:\n\
+		Summon angry wraiths which enact vengeance from beyond the grave on those still connected to this world.\n\
+		Note, that includes any of your vassals who are not undead, as wraiths will seek to kill them too!\n\
+		Summons more wraiths as this ability ranks up.\n\
+		These wraiths aren't very powerful, and best serve as a distraction, but in a pinch can help in a fight. \n\
+		The spirits will eventually return back to their realm if not otherwise destroyed."
+	check_flags = BP_CANT_USE_IN_TORPOR|BP_CANT_USE_IN_FRENZY|BP_CANT_USE_WHILE_UNCONSCIOUS
+	power_flags = BP_AM_STATIC_COOLDOWN
+	bloodcost = 15
+	cooldown_time = 60 SECONDS
+	///How many spirits should be summoned when cast
+	var/num_spirits = 1
+
+/datum/action/cooldown/bloodsucker/hecata/spiritcall/vassal //this variant has to exist, as hecata favorite vassals are technically in 'torpor'
+	check_flags = BP_CANT_USE_WHILE_INCAPACITATED|BP_CANT_USE_WHILE_UNCONSCIOUS
+
+/datum/action/cooldown/bloodsucker/hecata/spiritcall/ActivatePower(mob/user = owner)
```

Comment:
```suggestion
/datum/action/cooldown/bloodsucker/hecata/spiritcall/ActivatePower()
```
We shouldn't be doing this, use ``owner``, or typecast it in the proc if you need to call a proc on living/carbon/whatever